packages/core/src/forms/control-group/control-group.element.ts

```diff
@@ -105,9 +105,9 @@ export class CdsInternalControlGroup extends LitElement {
   /** @private */
   @query('cds-internal-control-label[action=primary]') controlLabel: CdsInternalControlLabel;
 
-  @query('slot[name=controls]') private controlSlot: HTMLElement;
+  @query('.controls') private controlSlot: HTMLElement;
 
-  @query('slot[name=message]') private messageSlot: HTMLElement;
+  @query('.messages') private messageSlot: HTMLElement;
```

Comment:
Since safari defaults slot to display:contents and ie11 polyfill removes the slot element its best to not use the slot as a selector in JS or Scss